dvc/command/experiments/utils.py

```diff
@@ -0,0 +1,32 @@
+def add_rev_selection_flags(
+    experiments_subcmd_parser, command: str, default: bool = True
+):
+    experiments_subcmd_parser.add_argument(
+        "-A",
+        "--all-commits",
+        action="store_true",
+        default=False,
+        help=(
+            f"{command} all experiments in the repository "
+            "(overrides `--rev`)."
+        ),
+    )
+    msg = f"{command} experiments derived from the specified Git commit"
+    if default:
+        msg += "(instead of `HEAD`)."
```

Comment:
Maybe we should hint more, i.e. "specified Git commit, branch, tag, etc." ?

Reply:
p.s. in docs (https://dvc.org/doc/command-reference/exp/show#-n) we say "from HEAD". That's incorrect right? Incomplete rather

Reply:
And here, actually I'm not sure we want conditional help output differences?

Reply:
Unresolving this one since there was no change (it's missing a period when `!default`) or answer.

Still wondering about the conditional help output though... Shouldn't it always be the static? (I'm not sure what the fn-level `default` var is for here.) Cc @dberenbaum

Reply:
Got it, thanks @karajan1001 !